FreeRTOS/Lab-Project-FreeRTOS-FAT
# Issues/ci build

Fix formating isssue in CI

Description
-----------
Formatting seems to have failed for a while: https://github.com/FreeRTOS/Lab-Project-FreeRTOS-FAT/actions/workflows/ci.yml

Pull request #40 is blocked due to failed CI step.

Uncrustified version used: Uncrustify-0.66.1_f

Uncrustified is now at version Uncrustify_d-0.75.1_f, could be an idea to update CI script. 

Test Steps
-----------
No test performed. Cosmetic change only.

Checklist:
----------

Only tested using test/build-combination/freertos_plus_fat_build_test  on GCC Posix.

Related Issue
-----------
PR #40 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
